components/compositing/webview_renderer.rs

```diff
@@ -742,9 +745,9 @@ impl WebViewRenderer {
         self.on_scroll_window_event(scroll_location, cursor)
     }
 
-    pub(crate) fn process_pending_scroll_events(&mut self, compositor: &mut IOCompositor) {
+    pub(crate) fn process_pending_scroll_events(&mut self) -> (bool, Option<ScrollResult>) {
```

Comment:
Might be good to have a doc comment explaining the meaning of the first component of the return value - whether the zoom has changed or not.

Reply:
Can we use an enum `ZoomResult` for the first component? I always find it confusing when there's an anonymous boolean.

And can we rename the function to `process_pending_scroll_zoom_events` for better clarity?

Reply:
Sure. I have renamed the function to `process_pending_scroll_and_pinch_zoom_events` and this now returns a `PinchZoomResult`.